dianatanase/assignment3/Q1_tests.java

```diff
@@ -0,0 +1,92 @@
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.junit.Test;
+
+
+public class Q1_tests {
+
+	// Set up the grid.
+	public char[][] gridSetUp(){
+		char[][] grid = new char[2][3];
+		grid[0][0] = 'a';
+		grid[0][1] = 'a';
+		grid[0][2] = 'r';
+		grid[1][0] = 't';
+		grid[1][1] = 'c';
+		grid[1][2] = 'd';
+		
+		return grid;
+	}
+		
+	// Set up a dictionary with a few words.
+	public Dictionary simpleDictionarySetUp(){
+		Dictionary dict = new Dictionary();
+		dict.addListOfWords(Arrays.asList("CAR", "CARD", "CART", "CAT", "c"));
+		
+		return dict;
+	}
+	
+	// Set up a dictionary with more words.
+	public Dictionary complexDictionarySetUp(){
+		Dictionary dict = new Dictionary();
+		dict.addListOfWords(Arrays.asList("CAR", "CARD", "CAT"));
+		dict.addListOfWords(Arrays.asList("dar", "aarct", "ata", "aar", "arc", "rdat", "rctaa", "racat", "draact"));
```

Comment:
If lists are getting really long like this, it can be neater to split them over multiple lines:
```
dict.addListOfWords(Arrays.asList(
    "dar", "aarct", "ata", "aar", "arc", "rdat", "rctaa", "racat", "draact"));
```